git_machete/cmd.py

```diff
@@ -1690,18 +1690,14 @@ def lb_is_not_b(lb: str, lb_or_rb: str) -> bool:
 # Complex routines/commands
 
 def is_merged_to(cli_ctxt: CommandLineContext, b: str, target: str) -> bool:
-    if commit_sha_by_revision(cli_ctxt, target) == commit_sha_by_revision(cli_ctxt, b):
-        # If branch is equal to the target, we need to distinguish between the
+    if is_ancestor_or_equal(cli_ctxt, b, target):
```

Comment:
The actual change... it's suspiciosly simple, I'm afraid this will backfire somehow that I can't see yet :/

Reply:
YOLO